packages/react-magma-dom/src/components/TimePicker/useTimePicker.ts

```diff
@@ -48,7 +48,12 @@ export function useTimePicker(props: UseTimePickerProps) {
   const id = useGenerateId(props.id);
 
   React.useEffect(() => {
-    if (validTime(props.value)) {
+    if(typeof props.value === 'undefined') {
+      setHour('');
+      setMinute('');
+      setAmPm('AM');
```

Comment:
We could use our `am` variable here for consistency 
```suggestion
      setAmPm(am);
```

Reply:
Thx! I will push changes now!